README.md

```diff
@@ -22,9 +22,8 @@ Civo CLI is a tool to manage your [Civo.com](https://www.civo.com) account from
 - [Load Balancers](#load-balancers)
 - [Quota](#quota)
 - [Sizes](#sizes)
-- [Snapshots](#snapshots)
 - [SSH Keys](#ssh-keys)
-- [Templates](#templates)
+- [DiskImages](#templates)
```

Comment:
The anchor link should be `#disk-image` here

Reply:
Got a request from @kunal-kushwaha to merge this as-is. He will do the fixes in another PR.

Reply:
Linking here for reference: https://github.com/civo/cli/pull/150